Stripe/STPAddressViewModel.m

```diff
@@ -93,16 +99,34 @@ - (instancetype)initWithRequiredShippingFields:(PKAddressField)requiredShippingA
     return self;
 }
 
+- (instancetype)initWithSEPADebitFields {
+    self = [super init];
+    if (self) {
+        _addressType = STPAddressTypeSEPADebit;
+        _addressCells = @[
+                          [[STPAddressFieldTableViewCell alloc] initWithType:STPAddressFieldTypeLine1 contents:@"" lastInList:NO delegate:self],
+                          [[STPAddressFieldTableViewCell alloc] initWithType:STPAddressFieldTypeCity contents:@"" lastInList:NO delegate:self],
+                          [[STPAddressFieldTableViewCell alloc] initWithType:STPAddressFieldTypeZip contents:@"" lastInList:NO delegate:self],
```

Comment:
Do we need to optionally hide postal code based on what country you have selected? Or do all SEPA countries have postal codes?